coop_membership/data/email_template_data.xml

```diff
@@ -272,10 +272,12 @@ If you think there was a mistake or wish to subscribe to another team please con
         <field name="name">Registration Confirm Meeting Email</field>
         <field name="model_id" ref="event.model_event_registration"/>
         <field name="email_from">${(object.company_id.email or '')|safe}</field>
-        <field name="email_to" >${object.email|safe}</field>
+        <field name="email_to">${object.email|safe}</field>
         <field name="lang">${(object.partner_id.lang)|safe}</field>
         <field name="reply_to">${object.company_id.email|safe}</field>
-        <field name="subject">Your registration to a welcome meeting on ${object.convert_event_date_begin()} at ${object.get_time_meeting()}</field>
+        <field name="subject">Your registration to a welcome meeting on ${object.convert_event_date_begin()} at
```

Comment:
```suggestion
        <field name="subject">Your registration to a welcome meeting on ${format_datetime(object.event_id.date_begin)}.
```

I think if we do this we can remove the `convert_event_date_begin()` and `get_time_meeting()` functions

Reply:
@ivantodorovich  : i tried `format_datetime` but gives an error  it is not  found but i think we can directly use `${object.event_id.date_begin_located}` it will display as below 
![date](https://user-images.githubusercontent.com/54934323/73905484-719b9400-48c5-11ea-80b5-beeb6d9b399a.png)

Reply:
Great!
Do it :+1: